controllers/updates/views/landing.njk

```diff
@@ -1,46 +1,66 @@
+                    {{ title }}
+                </a>
+            </h2>
+        </header>
+        <div class="latest-news__content">
+            <ul class="flex-grid flex-grid--3up">
+                {% for entry in entries | takeItems(3) %}
+                    <li class="flex-grid__item">
+                        {{ articleTeaser(entry) }}
+                    </li>
+                {% endfor %}
+            </ul>
+        </div>
+        <footer class="latest-news__footer">
+            <a class="latest-news__more" href="{{ allLink }}">
+                View all {{ title | lower }} &rarr;
+            </a>
+        </footer>
+    </section>
+{% endmacro %}
+
 {% block content %}
     {% set pageAccent = 'cyan' %}
 
     <main role="main" id="content">
         <section class="content-box u-inner-wide-only accent--{{ pageAccent }} a--border-top">
             {{ breadcrumbTrail(breadcrumbs) }}
 
-            {{ newsTitle(entriesMeta) }}
-
-            {% for type, entrylist in entries | groupItemsBy('updateType.name.plural') %}
-                {% set typeLink = localify('/news/' + entrylist[0].updateType.slug) %}
-                <section class="latest-news u-no-padding">
-                    <header class="latest-news__header">
-                        <h2 class="t1 t--underline a--text accent--blue">
-                            <a class="u-link-minimal" href="{{ typeLink }}">
-                                {{ type }}
-                            </a>
-                        </h2>
-                    </header>
-                    <div class="latest-news__content">
-                        <ul class="flex-grid flex-grid--3up">
-                            {% for entry in entrylist | takeItems(3) %}
-                                <li class="flex-grid__item">
-                                    {{ updateTeaser(entry) }}
-                                </li>
-                            {% endfor %}
-                        </ul>
-                    </div>
-                    <footer class="latest-news__footer">
-                        <a class="latest-news__more" href="{{ typeLink }}">
-                            View all
-                        </a>
-                    </footer>
-                </section>
-
-            {% endfor %}
+            <h1>{{ title }}</h1>
+
+            {# Updates #}
+            {{ newsSection(
+                title = copy.types.updates.plural,
+                entries = groupedEntries.updates,
+                allLink = localify('/news/updates')
+            )}}
+
+
+            {# Blog posts #}
+            {{ newsSection(
+                title = copy.types.blog.plural,
+                entries = groupedEntries.blog,
+                allLink = localify('/news/blog')
+            )}}
+
+            {# Press releases #}
+            {{ newsSection(
+                title = copy.types['press-releases'].plural,
+                entries = groupedEntries.press_releases,
```

Comment:
I think this will need to swap to `groupedEntries['press-releases'] once I've updated my version of the API.